_posts/2023-08-20-my-lfx-mentorship-experience-with-the-cncf-strimzi-community.md

```diff
@@ -0,0 +1,174 @@
+
+![Mentor's GitHub comment](/assets/images/posts/2023-08-20-github-comment.png)
+
+This scared me :) But I was confident I could provide a good proposal because of my previous experience with the underlying technologies.
+
+On the same day, I got my first interaction with my mentors, I would never think it wouldn’t be the last one.
+
+![Mentor's email asking for a proposal](/assets/images/posts/2023-08-20-mentor-email.png)
+
+The first task was understanding the real problem and why we would need to build such a bridge, so I researched that.
+To design the solution, my first intuition was to relate my college project to the project.
+Just to get a rough idea of how it is very similar to what I was building in college:
+
+![Architecture of a system using MQTT](/assets/images/posts/2023-08-20-design-of-a-system-using-mqtt.png)
+
+After that, I started to think about the architecture of the solution, and I came up with this:
+
+![Strimzi MQTT Bridge](/assets/images/posts/2023-08-20-mqtt-bridge-high-architecture.png)
+
+To be even closer to a logical solution, the win-win was to study and understand how Strimzi HTTP-Kafka Bridge works and how it is deployed on Kubernetes. 
+After reading the Strimzi HTTP-Kafka Bridge documentation, I was able to design a Bridge tackling all the issues. 
+Of course, it was not perfect, and this is where discussing about my solution with the mentors helped me.
+
+#### The mentors feedback
+
+After submitting my proposal, I got feedback from my mentors, and they were very positive about it.
+They had a lot of comments asking about my approach, and discussing and resolving the conversions was, of course, part of my win-win.
+At some point, I thought I wouldn’t be selected because my initial approach was mentioning unnecessary things like a full MQTT broker.
+
+The feedback from my mentors allowed me to rethink and defend my approach, proving that I had a basic understanding and was keen to work on the project.
+
+#### The great day
+
+After so many discussions on the right approach, feedback, etc., I was not sleeping waiting for my results. 
+My first result was for the CNCF- Landing Page Improvement, in which I got rejected :(. 
+
+I started running unfaithfully and waiting for the Strimzi result, but it was too late, so I fell asleep.
+Waking up too early, I opened the email, and the great news was there:
+
+![Acceptance email](/assets/images/posts/2023-08-20-acceptance-email.png)
+
+### The Mentorship journey
+
+Upon joining the Strimzi community, I was warmly welcomed by mentors who were experts in their fields and exceptionally supportive and approachable.
+With their sense of humor, they gave me comprehensive guidance on understanding the Strimzi project governance, setting up a repository for the project I worked on, etc.
+A big shout-out to [kyguy](https://github.com/kyguy) and [pppatierno](https://github.com/ppatierno) for being my mentors and guiding me throughout the mentorship journey.
```

Comment:
Too many "p" in my GitHub handle :-D ... the link is fine, the text should be "pp" and not "ppp" :-P

Reply:
Sorry :)